CHANGELOG.md

```diff
@@ -7,12 +7,14 @@
 ## Improvements
 
 - PR #144: Dockerfile update and docs for LinearRegression and Kalman Filter.
+- PR #168: Add /ci/gpu/build.sh file to cuML
 - PR #167: Integrating full-n-final ml-prims repo inside cuml
 - PR #114: Building faiss C++ api into libcuml
 - PR #64: Using FAISS C++ API in cuML and exposing bindings through cython
 
 ## Bug Fixes
 
+- PR #167: Integrating full-n-final ml-prims repo inside cuml
```

Comment:
Why is this change in this PR?

Reply:
Looks like it got duplicated in a conflict resolution since it is above already?